### PR TITLE
Lancedb Bug Fix and Configurations added

### PR DESCRIPTION
**Pull request by PrashantDixit0**
**Please describe the purpose of this pull request.**
This PR provides the options which will be required to be set configuration and small bug fix in #456

### CODE REVIEW

**Comment by PrashantDixit0** [comment]
Issue #456

File: README.md [hunk 1]
```diff
@@ -71,6 +71,18 @@ Configure default setting for MemGPT by running:
 ```
 memgpt configure
 ```
+Configurations to  be done:
```
Comment (cpacker) [inline README.md:74]: Can you remove this diff from the PR? Seems unrelated

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
+questionary==2.0.1
```
Comment (cpacker) [inline requirements.txt:1]: Can you remove this diff (.txt file) from the PR? We're using poetry for package management so if you want to add new packages, you need to run `poetry add` to do the dependency resolution.

We can also do this for you - you just need to tell us what extra packages this PR needs (which I don't think is any actually).

**Review by cpacker — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (PrashantDixit0) [reply requirements.txt:1]: Sure I'll remove it asap

File: README.md (hunk 1 above)
Comment (PrashantDixit0) [reply README.md:74]: Yes sure

**Comment by cpacker** [comment]
Potential related bug reported on discord but I haven't had a chance to debug myself yet:

image

https://discord.com/channels/1161736243340640419/1162177332350558339/1178076181136932884

**Comment by PrashantDixit0** [comment]
Done Changes

**Comment by cpacker** [comment]
@PrashantDixit0 can you add a section to the top-level post with instructions on:

1. How to reproduce the error that this PR patches (eg starting with `git checkout main` and `pip install -e .`, do `memgpt configure` with ...)
2. A statement / copy-paste block / screenshot showing what the new expected output / result is

I'll use (1) and (2) to check the PR then merge in.

**Comment by PrashantDixit0** [comment]
Screenshot from 2023-11-26 13-48-46
Screenshot from 2023-11-26 19-29-11

Here I have loaded documentation and it is working now

**Comment by PrashantDixit0** [comment]
I'll work on it and raise another PR for resolving it
